importlab/utils.py

```diff
@@ -59,6 +59,14 @@ def makedirs(path):
             raise
 
 
+def get_parent_directory(filenames):
```

Comment:
This doesn't seem to be used (or tested) anywhere.

Reply:
removed